.travis/travis-openmandriva.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+pushd $SCRIPT_DIR
+
+source $SCRIPT_DIR/travis-common.inc
+
+set -e
+set -x
+
+JOB_NAME=${TRAVIS_JOB_NAME:-OpenMandriva Lx cooker }
+
+arr=($JOB_NAME)
+release=${arr[1]:-cooker}
+
+mmd_run_docker_tests \
+    os=openmandriva \
```

Comment:
Take a look at the [opensuse](https://github.com/fedora-modularity/libmodulemd/blob/master/.travis/travis-opensuse.sh#L20) test.

When the image name has a forward slash in it, you need to add `image=openmandriva/$release` to the call and it should work.